runtime/binding-kafka-grpc/src/main/java/io/aklivity/zilla/runtime/binding/kafka/grpc/internal/stream/KafkaGrpcRemoteServerFactory.java

```diff
@@ -182,9 +182,12 @@ public void attach(
             newBinding.routes.forEach(r ->
                 r.when.forEach(c ->
                 {
+                    final String groupId = String.format("zilla-%s-%s", supplyNamespace.apply(binding.id),
```

Comment:
`KafkaConfiguration` changes look good for `supplyGroupId()`, but we want have it return the pattern and then substitute in the namespace and local name for the binding via `String.format` here.

The default pattern returned by `KafkaConfiguration.supplyGroupId()` would be `"zilla:%s-%s"`.